python/kfserving/kfserving/seldon.py

```diff
@@ -0,0 +1,55 @@
+from kfserving.protocol import ProtocolHandler
+import tensorflow as tf
+from tensorflow.core.framework.tensor_pb2 import TensorProto
+from google.protobuf import json_format
+import numpy as np
+import json
+
+class SeldonProtocol(ProtocolHandler):
+
+    def _extractTensor(self, body):
+        if "data" in body:
+            dataDef = body["data"]
+            if "tensor" in dataDef:
+                arr = np.array(dataDef.get("tensor").get("values")).reshape(dataDef.get("tensor").get("shape"))
+                return arr,"tensor"
```

Comment:
Maybe use enum instead of string for the type here?

Reply:
Can you expand on what you mean. The JSON dict has a set of values. Are you suggesting we create an enum with the possible values?

Reply:
string `tensor`, `ndarray` and `tftensor` are a bit overloaded here, one is input key and the other is for data type, how about create an enum `ProtocolDataType` so that user is clear what data types each protocol is supporting?

Reply:
Updated to use enum